platform/android/export/export.cpp

```diff
@@ -2136,6 +2179,40 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		Vector<String> enabled_abis = get_enabled_abis(p_preset);
 
+		String project_icon_path = ProjectSettings::get_singleton()->get("application/config/icon");
+
+		//Prepare images to be resized for the icons. If some image ends up being uninitialized,
```

Comment:
Not a big issue, but for comments we tend to favor this style:
```
// Starts with a space, capital letter and ends with a dot.
```
May not be used consistently in existing code though, but for new code we're trying to get more consistency.

Reply:
Done.